media-fonts/ibm_plex/ibm_plex-0.5.3.recipe

```diff
@@ -0,0 +1,34 @@
+SUMMARY="The package of IBM’s new typeface, IBM Plex"
+DESCRIPTION="Thanks for your interest in IBM Plex, IBM’s new corporate type family. \
```

Comment:
Can you use the wikipedia text here?

```
"IBM Plex is an open source typeface superfamily developed by Mike Abbink for IBM to use in branding. The typeface family was commissioned to substitute Helvetica Neue.
The family has three typefaces:
* IBM Plex Serif – A serif typeface.
* IBM Plex Sans – A sans-serif typeface.
* IBM Plex Mono – A monospaced typeface."
```

Reply:
Good catch! And good question too :)
I don't have the answer.
I think, however, that we should drop the "Thanks for your interest in IBM Plex" and even change the description.
@dcatt-haiku, would you agree with changing the `DESCRIPTION` to:
```
DESCRIPTION="IBM Plex is IBM’s new corporate type family. According to its \
owner, it can be used for all typographical situations, replacing Helvetica \
Neue, whenever possible."
```
If you would like to suggest something else, please feel free to do so.

Actually, I did not really look at this before my approval, but now I can see that it can't be merged with the current description.
Thanks!

Reply:
Oops, sorry @alaviss, I misunderstood your comment.

That's a very good idea, to say the font was developed by Mike Abbink for IBM to use in branding and to list the three typefaces.

Reply:
@alaviss, @dcatt-haiku, what do you think of:
```
DESCRIPTION="IBM Plex is IBM's new corporate type family. It is an open source \
typeface superfamily developed by Mike Abbink for IBM to use in branding. It \
was commissioned to substitute Helvetica Neue and has three typefaces:
* IBM Plex Serif – A serif typeface.
* IBM Plex Sans – A sans-serif typeface.
* IBM Plex Mono – A monospaced typeface."
```